src/main/default/classes/RatingsHandlerTest.cls

```diff
@@ -0,0 +1,23 @@
+@isTest
+private class RatingsHandlerTest {
+  @isTest
+  static void testGetRatingWithExistingRecord() {
+    Rating__c testRating = new Rating__c(
+      Score__c = 90,
+      Document_id__c = 'TestDocumentId'
+    );
+    insert testRating;
+
+    Rating__c result = RatingsHandler.getRating('TestDocumentId');
+
+    System.assertEquals(testRating.Id, result.Id, 'The Result Id received: ' + result.Id + ' was expected to be: ' + testRating.Id);
+    System.assertEquals(testRating.Score__c, result.Score__c, 'The Result score received: ' + result.Score__c + ' was expected to be: 90');
```

Comment:
Just to avoid hardcoding the value as a string.
```suggestion
    System.assertEquals(testRating.Score__c, result.Score__c, 'The Result score received: ' + result.Score__c + ' was expected to be: ' + testRating.Score__c);
```

Reply:
ok yeah didn't make much of it since it was the error message, but I guess it is best practice 😂